.vsts-ci.yml

```diff
@@ -4,7 +4,7 @@ variables:
 resources:
   containers:
   - container: ubuntu1404
-    image: lightgbm/vsts-agent:ubuntu-14.04
+    image: lightgbm/vsts-agent:ubuntu-14.04-dev
```

Comment:
This is to test and show that everything works.
I'll remove `dev` postfix before merging.